src/main/java/hudson/plugins/sonar/action/SonarBuildBadgeActionFactory.java

```diff
@@ -41,6 +42,9 @@
    * The badge will have an URL if there aren't multiple URLs.
    */
   public Collection<? extends Action> createFor(Run run) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Refactor this method to reduce its Cognitive Complexity from 11 to the 10 allowed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS3776)